src/en/reference-environment-variables.md

```diff
@@ -15,33 +15,22 @@ Use header tags so we can link to these variables individually.
 These variables are available on the juju client in order to change its default
 behavior.
 
-#### JUJU_ENV
 
-The current environment that juju will use by default. Juju has three ways it
-will  determine current environment, in descending order:
+#### JUJU_DATA
 
-1. Check to see if `JUJU_ENV` is set.
-```
-echo $JUJU_ENV
-local
-```
+This sets the path where Juju will look for its configuration files. You do not
+need to set this - by default Juju follows XDG guidelines and on Linux systems
+it will use the path:
 
-2. Read the `$JUJU_HOME/current-environment` file, which is set by the
-`juju switch` command.
-```
-juju switch amazon
-cat ~/.juju/current-environment
-amazon
+```no-highlight
+~/.local/share/juju
 ```
 
-3. Read `$JUJU_HOME/environments.yaml` and use the default environment.
-```yaml
-default: local
-```
+#### JUJU_HOME (Deprecated)
```

Comment:
I too appreciate a clean division of doc versions but we should be able to make exceptions. Due to the non-ready software we need to foresee what sort of information a user may want to look up and the JUJU_HOME variable is such a beast. The minor inconvenience in including 3 lines in the documentation far outweighs the frustration a user will likely feel by not including them. Since this is the only variable in this category, also clearly marked 'deprecated', it is indeed an exception.

I favour the inclusion of this variable in the 2.0 docs until the upgrade mechanism is built.